### ISSUE TITLE
Increase the minimum supported Android SDK version from 16 to 21

### ISSUE THREAD

**Issue by jackerley** [YoYoGames/GameMaker-Bugs#4459]
### Is your feature request related to a problem?

We're planning on making some changes in April that will be easier if we no longer need to support Android SDK versions below 21. Only a very small fraction of devices currently use API level <21 and Google Play Services already require a minimum of Android 5.0 (API level 21) Please let us know if this is going to cause a problem for you, or if there is any particular reason that you need to support devices with an API level of <21 so that we can consider alternatives if necessary.

### Describe the solution you'd like

Let us know now if this is going to cause you any problems.

**Comment by JujuAdams** [comment]
Netflix is running at API level 26. Other big mobile studios are running API level 22. I don't have all that much experience with embedded devices / TVs etc. but for mobile phones API level 21 is good.

**Comment by mmMarzban** [comment]
API 21 should be good.

**Comment by npapalia** [comment]
API 21 I believe should be fine as like stated GPS already requires a minimum of 21

**Comment by gnysek** [comment]
Level 21 = Android 5.0 ?

Kitkat was released nearly exactly 10 years ago (Feb '14), so we can say that everything before that is old enough now.

**Comment by FoxyOfJungle** [comment]
I'm fine with that. 21 is perfect.

**Comment by JonnoRobson** [comment]
The minimum SDK level for Android should now default to 21 rather than 16.

Additionally there is now a warning displayed when setting the target SDK level below API level 33 as per Google Play's recent target API changes: https://developer.android.com/google/play/requirements/target-sdk

**Comment by mgeddesGM** [comment]
verified as of  IDE v2024.400.0.541  Runtime v2024.400.0.560

**Comment by vogonp42** [comment]
I don't mind at all, since I'm not aiming to support devices with that SDK version anyway.  I am having issues with touch controls after setting up 21 as the minimum though.  I have a single button game, where the player object responds to a single tap.  Now there is no response to that action.   I tried setting up a test project, and I got the same result.  I reinstalled Android Studio, and the required dependencies, but that didn't seem to change anything.
